openedx/core/djangoapps/notifications/views.py

```diff
@@ -298,6 +298,10 @@ def get(self, request):
             count_total += count
             count_by_app_name_dict[app_name] = count
 
+        for app_name in COURSE_NOTIFICATION_APPS:
```

Comment:
I am not sure if it would work, as COURSE_NOTIFICATION_APPS is dict of dicts, so app_name would also be a dict 

i.e 
```python

COURSE_NOTIFICATION_APPS = {
    'discussion': {
        'enabled': True,
        'core_info': '',
        'core_web': True,
        'core_email': True,
        'core_push': True,
    }
}

for app_name in COURSE_NOTIFICATION_APPS:
    print(app_name)
   # it would print dict not "discussion" (str)

```